client/src/app/components/Layout/Layout.jsx

```diff
@@ -41,8 +48,23 @@ export class Layout extends Component {
     loginUser();
   };
 
+  displayModeSwitch = () => {
```

Comment:
I was thinking that this method was for switching between list and tile view.
It should be named like  isListView = () => 
beacuse this function returns bollean I think isListView is the best name for this.

then in the render method, you will have 
this.isListView && <Render Something>   and it is very clear that when it is a list view display something...otherwise not :)

Reply:
I agree that this method name is unclear but this method doesn't switch between list and tiles views. Its duty is to check if the "view-switching" button should be displayed (this button should be displayed on the dashboard and on cohort view only because only on those two views lists are displayed). I moved the toolbar to the layout folder (just like you did in your task - on my demand of course). Maybe it was the wrong idea. Now, in this method I check the current route and base on this checking decide if this button should be displayed - handlers stay in layout component. I can move toolbar back (to main module's components) and pass variables and all handlers via props to dashboard/cohort. I changed this method name to `isViewSwitchDisplayed`

Reply:
No leave the toolbar where it is.

I know that this method is for checking if the switch button should be displayed but he name is to wrong and now even to complicated.

take a look, what this method is doing??  It's is checking if current path is LIST ? Am I right?

And later in render function 
`If(isListView) --> display switch button `  it's clear and easy to read for everybody from outside the project 

isViewTypeSwitchDisplayed would be a method to check if the button is displayed, rendered or something else. 

You are checking inside this method if path is /list and the method name is isViewTypeSwitchDisplated - does it make sense to you?